example/gin/go.sum

```diff
@@ -39,6 +39,7 @@ github.com/google/go-cmp v0.5.5/go.mod h1:v8dTdLbMG2kIc/vJvl+f65V22dbkXbowE6jgT/
 github.com/google/gofuzz v1.0.0/go.mod h1:dBl0BpW6vV/+mYPU4Po3pmUjxk6FQPldtuIdl/M65Eg=
 github.com/instana/go-sensor v1.55.0 h1:9Dpo0S9hah1irJAkkpGMfiAoKMbLLOtYBbtdhJbGvUM=
 github.com/instana/go-sensor v1.55.0/go.mod h1:19yQd89yv2d0O2+onnGL5WvtMS5c/HVzl14ko8eZgqo=
+github.com/instana/go-sensor v1.56.0/go.mod h1:Ks06EG9Da5O3hbdJiHIePG/vNmToovkaJjMlUBd70Yc=
```

Comment:
Not a huge deal, but if you run `go mod tidy` the old unused hash for v1.55.0 will disappear.

Reply:
There are lot of examples which are still depending on older versions of go-sensor. I just updated a few which my IDE reported. I am now updating the go.sum also. But additional cleanup can be tracked in another PR in my opinion.

Reply:
Sure, I don't think this is a big deal, as I said.
We can do this at any other time :)